### ISSUE TITLE
Script does not work if Proton is installed in a different steam library

### ISSUE THREAD

**Issue by sanyarajan** [ninelore/armaonlinux#2]
The script assumes that Proton is always installed in the .local/share/Steam library.

I had mine installed in a different library and had to change the line that set PROTONEXEC to point to the right path.

However, after changing the line to:
`PROTONEXEC="/media/storage/SteamLibrary/steamapps/common/Proton\ $PROTON_OFFICIAL_VERSION/proton"`

I get the following error when trying the install:
```
./Arma3TS_1v14-3.sh install Downloads/TeamSpeak3-Client-win64-3.5.6.exe
Trying to install Teamspeak with provided file
INSTALL TEAMSPEAK FOR ALL USERS AND LEAVE THE PATH DEFAULT!!!
Traceback (most recent call last):
  File "/media/storage/SteamLibrary/steamapps/common/Proton 6.3/proton", line 1031, in <module>
    g_session.init_session(sys.argv[1] != "runinprefix")
  File "/media/storage/SteamLibrary/steamapps/common/Proton 6.3/proton", line 860, in init_session
    g_compatdata.setup_prefix()
  File "/media/storage/SteamLibrary/steamapps/common/Proton 6.3/proton", line 483, in setup_prefix
    steamdir = os.environ["STEAM_COMPAT_CLIENT_INSTALL_PATH"]
  File "/usr/lib/python3.9/os.py", line 679, in __getitem__
    raise KeyError(key) from None
KeyError: 'STEAM_COMPAT_CLIENT_INSTALL_PATH'
```

This is the debug output:
```
./Arma3TS_1v14-3.sh debug
DEBUGGING INFORMATION
Command Line:
sh -c "/media/storage/SteamLibrary/steamapps/common/Proton\ 6.3/proton run /media/storage/SteamLibrary/steamapps/compatdata/107410/pfx/drive_c/Program\ Files/TeamSpeak\ 3\ Client/ts3client_win64.exe"

Proton: official Protn 6.3

Enviromentals were successfully set

STEAM_COMPAT_DATA_PATH: /media/storage/SteamLibrary/steamapps/compatdata/107410
SteamAppId/SteamGameId: 107410 107410
LD_PRELOAD: 
ESync: false
FSync: false
```

**Comment by ninelore** [comment]
Someone reported me a similar Issue some time ago on the ArmaOnUnix Discord. This is very ljkely an upstream issue that will be almost surely not be fixed due to the fact that proton is more than wine and not designed to run anything non-steam. 

For now i can only advise to have Proton in your main Steam folder.

Its also possible that the Steam devs intent to have proton only in the main Steam folder, but thats another story.

**Comment by sanyarajan** [comment]
I tried moving the Proton install to the default location and I still got the same error. I was able to install it by using protontricks to run explorer in the Arma3 prefix and starting the installer from there.

The script gives the same error when trying to start Teamspeak though. 

I found a way to get it to launch correctly using steamtinkerlaunch (https://github.com/frostworx/steamtinkerlaunch).

Thank you for your work on the guide it has helped me a lot to get everything set up
